src/publish.js

```diff
@@ -359,7 +359,7 @@ have published it.\
           this.spawn(gitCommand, ['commit', '-m', message], (code, stderr, stdout) => {
             stderr ??= '';
             stdout ??= '';
-            if (code === 0) {
+            if (code !== 0) {
```

Comment:
Kind of odd this was wrong, I believe it was inadvertently changed as part of the _utterly massive_ promisification PRs, from looking at the blame history.

This change seems legit. All other calls to `logFailure()` are when `code !== 0`, as in error code rather than clean exit code. So yeah, seems legit IMO.

Reply:
Was surprised to see this one broken for so long tbh